common/buildcraft/lib/net/MessageUpdateTile.java

```diff
@@ -46,6 +49,7 @@ public void fromBytes(ByteBuf buf) {
     public void toBytes(ByteBuf buf) {
         PacketBuffer buffer = new PacketBuffer(buf);
         buffer.writeBlockPos(pos);
+        buffer.writeString(className);
```

Comment:
Hm... We can use tile.getClass().getName().hashCode() for auto generating this number, but this is a bug, and we need to fix it. This is a hack.